pkgs/development/python-modules/pydiscourse/default.nix

```
@@ -0,0 +1,34 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub ,
+  lib,
+  pythonAtLeast,
+  requests,
+}:
+
+buildPythonPackage rec {
+  pname = "pydiscourse";
+  version = "1.4.0";
+  format = "setuptools";
+
```

Supporting EOL releases Python doesn't make much sense for new packages as the chance is high that nobody will test it. Upstream still mention an [addition requirement](https://github.com/pydiscourse/pydiscourse/blob/master/setup.cfg#L21C1-L21C33) for Python < 3.6 but the classifiers only mention Python 3.7.

```suggestion

  disabled = pythonOlder "3.7";
```